### PR TITLE
jenkins_build.sh: Upload to dockerhub resinhup packages for productio…

### PR DESCRIPTION
**Pull request by agherzan**
…n builds

Signed-off-by: Andrei Gherzan <andrei@resin.io>

Connects to resin-os/resinhup#28

### CODE REVIEW

**Comment by floion** [comment]
LGTM

File: automation/jenkins_build.sh [hunk 1]
```diff
@@ -126,6 +126,24 @@ cp $DEVICE_TYPE_JSON $BUILD_DEPLOY_DIR/device-type.json
 # move to deploy directory the kernel modules headers so we have it as a build artifact in jenkins
 mv -v $WORKSPACE/build/tmp/deploy/images/$MACHINE/kernel_modules_headers.tar.bz2 $BUILD_DEPLOY_DIR
 
+# If this is a clean production build, push a resinhup package to dockerhub
+if [[ "$sourceBranch" == production* ]] && [ "$metaResinBranch" == "__ignore__" ] && [ "$supervisorTag" == "__ignore__" ]; then
+    echo "INFO: Pushing resinhup package to dockerhub."
+    SLUG=$(jq --raw-output '.slug' $DEVICE_TYPE_JSON)
+    DOCKER_REPO="resin/resinos"
+    DOCKER_TAG="resinhup-$SLUG-$VERSION_HOSTOS"
```
Comment (petrosagg) [inline automation/jenkins_build.sh]: Shouldn't the tag be just `$VERSION_HOSTOS-$SLUG`?

Comment (agherzan) [inline automation/jenkins_build.sh]: This is the name format I used in resinhup too and they need to match.

Comment (petrosagg) [inline automation/jenkins_build.sh]: ok, let's change it there too. I don't think we'll ever have a non-resinhup ResinOS docker image. We're pushing towards unification

Comment (agherzan) [inline automation/jenkins_build.sh]: Ok. Changed.

**Pull request merged**